### PR TITLE
Add more metadata to the report attributes

### PR DESCRIPTION
**Pull request by hughsie**
This can be used in gnome-control-center when building the security report.

Fixes https://github.com/fwupd/fwupd/discussions/5262

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
Urh, contrib/codespell.cfg is being fixed by... codespell...

File: src/fu-engine.c [hunk 1]
```diff
@@ -2006,6 +2006,28 @@ fu_engine_get_boot_time(void)
 	return NULL;
 }
 
+static void
+fu_engine_get_report_metadata_cpu_device(FuEngine *self, GHashTable *hash)
+{
+	g_autofree gchar *guid = fwupd_guid_hash_string("cpu");
```
Comment (superm1) [inline src/fu-engine.c:2012]: This feels like it's getting too specifically into the current situation of some older CPUs don't export the needed information.

I feel we should be sticking to whether ANY plugin doesn't export enough information to better scale for future measurements.

Comment (hughsie) [reply src/fu-engine.c:2012]: This is getting the device with that attribute, not the security attribute. i.e. it's the data from CPUID.

Comment (superm1) [reply src/fu-engine.c:2012]: Oh right okay. No worry then.

**Pull request merged**